generators/search-map.js

```diff
@@ -21,14 +24,17 @@ var fs = require('fs'),
 module.exports = function(docMap, siteConfig) {
 	var searchMap = {},
 		name;
-
 	for (name in docMap) {
 		if (docMap.hasOwnProperty(name)) {
 			var docObj = docMap[name];
+			var description = markdownParser.parse(docObj.description);
+			description = markdownRenderer.render(description);
+			description = striptags(description);
```

Comment:
If I understand this correctly, it removes any html tags from the description. I think it’d be nice if the search results had the same markup as what’s on the page. Are there any technical reasons why we can’t leave them in?